classes/Product.php

```diff
@@ -962,6 +962,22 @@ public static function updateIsVirtual($id_product, $is_virtual = true)
         ], 'id_product = ' . (int) $id_product);
     }
 
+    /**
+     * @see ObjectModel::validateField()
+     *
+     * reset static cache (eg unit testing purpose).
+     */
+    public static function resetStaticCache()
```

Comment:
😱

Reply:
Yes indeed ^^ There were some problems with behat suites, the price of a product was static cached (based on its id) So it used the old price when another suite started (DB reset, so same id, the price came from the cache..)
This is really a pain 😅

Reply:
This kind of method is used in many other objectmodel classes (`resetStaticCache` or `resetCache`)

Reply:
Yep it actually does the same thing as `ObjectModel::resetStaticCache` but with more clearing related to Product cache only
Which makes me realize I need to fix the PHPDoc here